### ISSUE TITLE
incorrect classification of rain vs showers

### ISSUE THREAD

**Issue by level20peon** [open-meteo/open-meteo#962]
After issue #935 there's still some inconsistency of the classification of the precipitation details. 

I am not selecting a specific model, but according to the reply from @patrick-zippenfenig in #935, the KNMI Harmonie is used for my location (lat 50.7766, lon 6.0834).

If you use this location as coordinates [here](https://open-meteo.com/en/docs/historical-forecast-api#start_date=2024-08-17&end_date=2024-08-17&hourly=precipitation,rain,showers,snowfall), you will get a render like this:
![image](https://github.com/user-attachments/assets/3aea747c-3306-430c-ad00-b3c60849da25)

As you can see, all of the precipitation is classified as showers, but we even had severe weather alerts for that day...
`Es tritt Dauerregen auf. Dabei werden Niederschlagsmengen zwischen 25 l/m² und 40 l/m² erwartet. In Staulagen werden Mengen um 50 l/m² erreicht. - Gefahr durch vereinzelte Überflutungen von Straßen/Unterführungen.
Handlungsempfehlungen: Verhalten im Straßenverkehr anpassen; überflutete Abschnitte meiden`

... and thus most if not all of the precipitation yesterday was definitely rain, not showers. The _DWD ICON D2_ classified the precipitation type correctly for that time frame again.

This was one concrete example, not just one outlier of one day since issue #935 was closed.

So this made me wonder if you could add the following algorithm to the selected weather model system:
1) if any specific model is picked (I would pick _DWD ICON D2_ for example)
2) fill the response for the requested time frame with data from that specific model
3) "pad" any missing data with the _Best match_ model

I am asking this because if I select _DWD ICON D2_, a lot of the data I request only has forecast entries for a couple of hours/days but not for the whole requested time frame.

**Comment by patrick-zippenfenig** [comment]
Yes, this location is now using HARMONIE forests from KNMI. To get ICON-D2 forecasts and combine it with other ICON domains to get a 7 day forecast, you can select [`DWD ICON Seamless`](https://open-meteo.com/en/docs#hourly=temperature_2m,precipitation,rain,showers&past_days=7&models=icon_seamless).

Please note that weather models split between showers and large scale rain due to the way they calculate precipitation. Most weather models are not capable of modelling convection and showers directly (see convection permitting models). As a workaround they estimate convective precipition (showers) with some heuristics. You cannot compare this directly to a synoptic weather forecast from a meteorologist.

I would absolutely not relay on the differentiation between showers and rain from weather models. This is merely an indication from which "part" of a weather model this precipitation is calculated.

**Comment by level20peon** [comment]
ok, I see.

So how do meteorologists go from weather models to "their" forecast in terms of precipitation classification? Is there a way to imitate/calculate this transformation?

Also, I was just assuming that the _DWD Icon_ model was used for my coordinates before, but I am not sure, since I mostly used _Best match_. Could you please confirm that the _DWD Icon Seamless_ model was used for my coordinates before the _KNMI_ models were integrated (if not _DWD Icon_, which one was the selected model for my case)?
